src/services/dynamo-database-service.ts

```diff
@@ -145,6 +144,6 @@ export class DynamoDatabaseService {
       logAndPublishMetric(MetricNames.DB_QUERY_ERROR_TOO_MANY_ITEMS);
       throw new TooManyRecordsError(errorMessage);
```

Comment:
This isn't part of the PR, and I might be missing something, but why are we throwing a 'too many records' error here?

Reply:
Because we only ever expecting a given user to have one row in our DB, so if multiple records are returned there is something wrong with that user / our solution